client/my-sites/domains/domain-management/edit/transfer.jsx

```diff
@@ -62,6 +62,22 @@ class Transfer extends React.PureComponent {
 				);
 			}
 		} else {
```

Comment:
This probably needs a condition to check for the `transferStatus.CANCELLED` status - otherwise, we show the failure notice for the `PENDING_OWNER` status:

<img width="747" alt="Screenshot 2019-11-13 at 23 13 10" src="https://user-images.githubusercontent.com/3392497/68812558-48bf8600-066b-11ea-80f8-adc230b67f96.png">